packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -61,6 +76,38 @@ export type UseResizeHandleParams = {
   relative?: boolean;
 };
 
+const getUpdateEventData = (
+  event: NativeTouchOrMouseEvent | KeyboardEvent | CustomEvent,
+  value: number
+): ResizeHandleUpdateEventData => {
+  switch (event.constructor) {
+    case TouchEvent:
```

Comment:
We can get a problem as there similar to this one:

https://github.com/microsoft/fluentui/blob/5c671b32c10347f647b09105695f840824ebee4e/packages/react-components/react-utilities/src/utils/isHTMLElement.ts#L2-L6

I.e. could be broken for multi `window` scenarios as `TouchEvent` is actualy `window.TouchEvent`

Reply:
okay I have an idea how to fix this without the construtor